### ISSUE TITLE
[Screen reader - Bot Framework Web Chat - Card Breakfast]: Screen reader is announcing incorrect position information as '2 of 3' list items for the 'Snooze time' combo box.

### ISSUE THREAD

**Issue by KattaBhargav** [microsoft/BotFramework-WebChat#5357]
### Is it an issue related to Adaptive Cards?

No.

### User Experience:
If the screen reader announces does not announce position count or incorrect position count for the controls, then the person who relies on screen reader will not be able get the correct information to activate and use the controls.
### Note: 
User credentials should NOT be included in the bug.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### How to reproduce the issue?

1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3. Navigate to white label experience chat page and type Card Breakfast word in type your message text box.
4. Navigate to the results.
5. Verify whether screen reader announcing correct position or not.

### What do you expect?

Screen reader should announce correct position information for the 'Snooze time' combo box.
**Ex:** Screen reader should announce as 'Snooze time 10 minutes combo box collapsed has pop up' after enter key '10 minutes 1 of 2 selected'

### What actually happened?

Screen reader is announcing incorrect position information as '2 of 3' list items for the 'Snooze time' combo box.
**Observation:** Screen reader is announcing as 'Snooze time 10 minutes combo box collapsed has pop up' after enter key '10 minutes 1 of 3 selected'

### Do you have any screenshots or recordings to repro the issue?

https://github.com/user-attachments/assets/43894366-93ab-4587-bf9b-7a33aeb4abcc
![Image](https://github.com/user-attachments/assets/28b023b9-eb70-4a76-9430-158a938ddcde)

### MAS reference

MAS 1.3.1 – Info and Relationships

**Comment by KattaBhargav** [comment]
#A11yMAS;#A11ySev3;#HCL;#Accessibility;#AILimited;#Benchmark;#BotFrameworkWebChat-Oct24;#MAS1.3.1;#Win11-Edge(Chromium);

**Comment by Kirank915** [comment]
@OEvgeny This issue is marked as external to Adaptive cards, could you please confirm if your team will fix the issue, or we need to share this issue to adaptive cards team?

**Comment by OEvgeny** [comment]
This one is on adaptive cards. See https://github.com/microsoft/AdaptiveCards/issues/9022
